Cloud Application Manager/Core Concepts/providers.md

```diff
@@ -100,21 +100,21 @@ Cloud Application Manager integrates with many popular clouds. Here is a full li
 
 | Clouds                                                                                                                | Enterprise <br> (Saas) | Enterprise <br> (Appliance) |
 | --------------------------------------------------------------------------------------------------------------------- | :--------------------: | :-------------------------: |
-| [CenturyLink Cloud](../Deploying Anywhere/using-centurylink-cloud.md)                                               | ✓                      | ✓                           |
-| [CenturyLink DCC](../Deploying Anywhere/using-dcc.md)                                                               | ✓                      | ✓                           |
-| [CenturyLink Private Cloud on VMware Cloud Foundation](../../CenturyLink Private Cloud on VMware Cloud Foundation/General/#1) | ✓                      | ✓                           |
-| [Amazon Web Services](../Deploying Anywhere/using-your-aws-account.md)                                              | ✓                      | ✓                           |
-| Amazon Web Services GovCloud (Same documentation as [Amazon Web Services](../Deploying Anywhere/using-your-aws-account.md)) | ✓              | ✓                           |
-| [CloudStack](../Deploying Anywhere/using-cloudstack.md)                                                             | ✓                      | ✓                           |
+| [CenturyLink Cloud](../Deploying%20Anywhere/using-centurylink-cloud.md)                                               | ✓                      | ✓                           |
+| [CenturyLink DCC](../Deploying%20Anywhere/using-dcc.md)                                                               | ✓                      | ✓                           |
+| [CenturyLink Private Cloud on VMware Cloud Foundation](../../Deploying%20Anywhere/using-private-cloud-on-vmware-cloud-foundation.md) | ✓                      | ✓                           |
+| [Amazon Web Services](../Deploying%20Anywhere/using-your-aws-account.md)                                              | ✓                      | ✓                           |
+| Amazon Web Services GovCloud (Same documentation as [Amazon Web Services](../Deploying%20Anywhere/using-your-aws-account.md)) | ✓              | ✓                           |
+| [CloudStack](../Deploying%20Anywhere/using-cloudstack.md)                                                             | ✓                      | ✓                           |
```

Comment:
Please change all `%20` back to an actual space ` `. 
These links are to a relative file, not http links.  Even with http links, the system will handle the conversion, so no need for encoding.